debian/bullseye/8.0/Dockerfile

```diff
@@ -126,13 +134,23 @@ ENV NEWRELIC_DISTRIBUTED_TRACING_EXCLUDE_NEWRELIC_HEADER=0
 
 RUN bash -c 'if [ "${TARGETPLATFORM}" == "linux/amd64" ]; then \
   curl -L "https://download.newrelic.com/php_agent/archive/${NEWRELIC_VERSION}/newrelic-php5-${NEWRELIC_VERSION}-linux.tar.gz" | tar -C /tmp -zx \
-  && export NR_INSTALL_USE_CP_NOT_LN=1 \
-  && export NR_INSTALL_SILENT=1 \
-  && /tmp/newrelic-php5-*/newrelic-install install \
-  && rm -rf /tmp/newrelic-php5-* /tmp/nrinstall* \
-  && /usr/bin/install -d -m 777 /var/log/newrelic/ \
-  && rm /usr/local/etc/php/conf.d/newrelic.ini \
-  && ln -s /lib64/ld-linux-x86-64.so.2 /lib/ld-linux-x86-64.so.2; fi'
+    && export NR_INSTALL_USE_CP_NOT_LN=1 \
+    && export NR_INSTALL_SILENT=1 \
+    && /tmp/newrelic-php5-*/newrelic-install install \
+    && rm -rf /tmp/newrelic-php5-* /tmp/nrinstall* \
+    && /usr/bin/install -d -m 777 /var/log/newrelic/ \
+    && rm /usr/local/etc/php/conf.d/newrelic.ini \
+    && ln -s /lib64/ld-linux-x86-64.so.2 /lib/ld-linux-x86-64.so.2; fi'
+
+COPY --from=spyker-newrelic-arm /newrelic-daemon /tmp/newrelic-daemon
+
+RUN bash -c 'if [ "${TARGETPLATFORM}" == "linux/arm64" ]; then \
```

Comment:
Can we handle this staff on the `spyker-newrelic-arm` side?

Reply:
Nope. I tried to use an agent from `spyker-newrelic-arm` based on Aws Linux and Debian. And every time, we had an error with `libpcre`. For Debian, I found this solution: compile and install the agent directly with `make agent-install`